pkg/hintrunner/zero/zerohint_math.go

```diff
@@ -711,4 +711,59 @@ func createUnsignedDivRemHinter(resolver hintReferenceResolver) (hinter.Hinter,
 	}
 
 	return newUnsignedDivRemHinter(value, div, q, r), nil
+
+}
+
+func newIsQuadResidueHint(x, y hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "IsQuadResidue",
```

Comment:
Is this the right name for the function? Looking at other VMs, it seems it should be called `recoverY` from which given a known x coordinante of a point in the elliptic curve you find the y corrdinate.

Reply:
I am mistaken, after looking more, it is called like this.